cmd/server/main.go

```diff
@@ -26,6 +26,16 @@ func main() {
 			Value: "/var/lib/ocid/sandbox",
 			Usage: "ocid pod sandbox dir",
 		},
+		cli.Float64Flag{
```

Comment:
Yeah, we should check the config and fail if required but we shouldn't need to pass them from here.